src/bin/vip-cache-purge.js

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import { appQuery, purgeCache } from 'lib/api/cache-purge';
+import command from 'lib/cli/command';
+import { trackEvent } from 'lib/tracker';
+import * as exit from 'lib/cli/exit';
+
+const examples = [
+	{
+		usage: 'vip @123.production cache purge <URL1> <URL2>',
```

Comment:
Is this how we want to support purging multiple URLs? Or would it be better to allow passing in multiple URLs via `stdin` or a text file?

Reply:
@chriszarate Do you have an example with stdin or text file working? I've tried a few things but with no success